Cannot Print Config

Recently I got the following exception:

```
➜  kaicode.github.io git:(mipo256) aidy config
2025/06/02 13:56:32 execute (/Users/lombrozo/Workspace/OpenSource/kaicode.github.io): "git --version"
2025/06/02 13:56:32 execute (/Users/lombrozo/Workspace/OpenSource/kaicode.github.io): "git rev-parse --show-toplevel"
2025/06/02 13:56:32 execute (/Users/lombrozo/Workspace/OpenSource/kaicode.github.io): "git rev-parse --show-toplevel"
2025/06/02 13:56:32 Deepseek key is found
Aidy Configuration:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0xb0c2315]

goroutine 1 [running]:
github.com/volodya-lombrozo/aidy/internal/aidy.(*real).PrintConfig(0xc00017f500)
        /Users/lombrozo/Workspace/OpenSource/aidy/internal/aidy/real.go:189 +0x35
github.com/volodya-lombrozo/aidy/cmd.newRootCmd.newConfigCmd.func8(0xc000132e00?, {0xb1240ca?, 0x4?, 0xb123ffe?})
        /Users/lombrozo/Workspace/OpenSource/aidy/cmd/config.go:13 +0x1f
github.com/spf13/cobra.(*Command).execute(0xc00018d808, {0xb4e1480, 0x0, 0x0})
        /Users/lombrozo/go/pkg/mod/github.com/spf13/cobra@v1.9.1/command.go:1015 +0xa94
github.com/spf13/cobra.(*Command).ExecuteC(0xc00018c008)
        /Users/lombrozo/go/pkg/mod/github.com/spf13/cobra@v1.9.1/command.go:1148 +0x40c
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/lombrozo/go/pkg/mod/github.com/spf13/cobra@v1.9.1/command.go:1071
github.com/volodya-lombrozo/aidy/cmd.Execute()
        /Users/lombrozo/Workspace/OpenSource/aidy/cmd/root.go:13 +0x1f
main.main()
        /Users/lombrozo/Workspace/OpenSource/aidy/main.go:11 +0x13
```

It is definitely a bug

--- a/internal/aidy/real.go
+++ b/internal/aidy/real.go
@@ -52,6 +52,7 @@ func NewAidy(summary bool, aider bool, ailess bool) Aidy {
 	conf := newconf(aider, aidy.git)
 	aidy.ai = brain(ailess, summary, conf)
 	aidy.github = newgithub(aidy.git, conf, aidy.cache)
+	aidy.config = conf
 	aidy.initSummary(summary)
 	return &aidy
 }

